CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
```

Comment:
Maybe very nitpicking but I think the best label would be `Fixed`

Reply:
When it comes to your opinion it never is nitpicking <3